lib/Doctrine/DBAL/Connection.php

```diff
@@ -1334,6 +1336,8 @@ private function commitAll()
     /**
      * Cancels any database changes done during the current transaction.
      *
+     * @return bool
+     *
```

Comment:
Why not use `void` here? There is no `return false`, failures are conveyed with exceptions

Reply:
Because it implements `Doctrine\DBAL\Driver\Connection::rollBack() : bool`. The interface an the implementation were reworked in https://github.com/doctrine/dbal/pull/3480.